packages/components/src/Components/TagModal/TagModal.js

```diff
@@ -86,6 +87,15 @@ export default class TagModal extends React.Component {
                     ]
                 }}
             >
+                <ChipGroup>
+                    {selected?.map(({ id, key }) => <Chip
```

Comment:
just a nitpick: maybe we could refactor this to 
```jsx
{selected.length > 0 &&  <ChipGroup>
     {selected.map( ....
```

to not show ChipGroup when no chip is selected.

Reply:
`selected?.map` should do the trick